### ISSUE TITLE
Invalid byte sequence

### ISSUE THREAD

**Issue by skatenerd** [Gabriel439/Haskell-Turtle-Library#319]
When I try to download a file that contains an "ñ" character, I get an invalid byte sequence error:

`view $ inproc "curl" ["https://www2.census.gov/geo/relfiles/cdsld13/35/co_cd_35.txt"] empty`

Is there a way to encourage this to somehow start using unicode?

**Comment by Gabriel439** [comment]
@skatenerd: Yes.  You can use the following command to specify that your program should assume a UTF-8 encoding globally:

```
GHC.IO.Encoding.setLocaleEncoding System.IO.utf8
```

**Comment by Gabriel439** [comment]
@skatenerd: You're welcome! :slightly_smiling_face:
